src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ResolvedSdkReference.xaml

```diff
@@ -8,17 +8,18 @@
     xmlns="http://schemas.microsoft.com/build/2009/properties">
     <Rule.DataSource>
         <DataSource Persistence="ResolvedReference" ItemType="SDKReference" HasConfigurationCondition="False"
-                    SourceType="TargetResults" MSBuildTarget="ResolveSDKReferencesDesignTime" SourceOfDefaultValue="AfterContext" />
+                    SourceType="TargetResults" MSBuildTarget="CollectResolvedSDKReferencesDesignTime" />
     </Rule.DataSource>
     <StringProperty Name="AppXLocation" DisplayName="App Package Location" />
-
     <!-- This property should be the same as the one for the ResolvedReference item -->
     <StringProperty Name="OriginalItemSpec" Visible="false" />
-
+    <StringProperty Name="SDKRootFolder" DisplayName="SDK Root" />
```

Comment:
yes translation is needed and i am runing the script now. However visible should not be false - we do want to show Name, Version and Path (SDK Root in this case)

---
In reply to: [98366079](https://github.com/dotnet/roslyn-project-system/pull/1346#discussion_r98366079) [](ancestors = 98366079)